packages/freetype.rb

```diff
@@ -3,40 +3,44 @@
 class Freetype < Package
   description 'FreeType is a freely available software library to render fonts.'
   homepage 'https://www.freetype.org/'
-  version '2.10.4'
+  version '2.11.0'
   license 'FTL or GPL-2+'
-  compatibility 'all'
-  source_url 'https://download.savannah.gnu.org/releases/freetype/freetype-2.10.4.tar.xz'
-  source_sha256 '86a854d8905b19698bbc8f23b860bc104246ce4854dcea8e3b0fb21284f75784'
+  compatibility 'x86_64'
```

Comment:
Are you sure this is incompatible with arm?

Reply:
Sorry that's my build script automatically setting that based upon what binaries were found locally. The next upload will fix this.